_reference/validation_errors.md

```diff
@@ -209,12 +208,11 @@ all attributes with a prefix of `"data-"` are also whitelisted.
   </tr>
 </table>
 
-Cdata is the content data between a start and end HTML tag
+CDATA is the content data between a start and end HTML tag
 and is currently evaluated with both whitelists and blacklists.
-Tags with mandatory cdata include:
+Tags with mandatory CDATA include:
 
-* `<style amp-boilerplate>body{...}</style>`
-* `<noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>`
+* `<style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>`
```

Comment:
nit: Maybe use this form
```markdown
 ``.`html
  <html>
  `.``
```
For better syntax highlighting? (Remote dots; just added for encoding reasons).